example/src/App.tsx

```diff
@@ -1,10 +1,22 @@
 import * as React from 'react';
 
+import { useEffect } from 'react';
+
 import { Button, View } from 'react-native';
 import { type AppViewInterface, appViews } from './AppViewInterface';
 import { styles } from './Styles';
+import { Linking } from 'react-native';
 
 export default function App() {
+  useEffect(() => {
+    Linking.addEventListener('url', ({ url }) => {
+      console.log('Event Listener: url', url);
```

Comment:
I like this for now - maybe in the future we could add some kind of navigation.

Reply:
Might be tricky in the example app though.

Reply:
Might be possible since it's just navigating within RN. Let me see if I can find a quick example. Good idea though.

Reply:
Going to leave it as is for now and cut out a task to handle this later as it looks a little more involved.

Reply:
Yah that makes more sense to me. This is good for now!